app/views/contexts/summary/block.py

```diff
@@ -21,14 +23,24 @@ def __init__(
         self.title = block_schema.get("title")
         self.number = block_schema.get("number")
         self.link = self._build_link(block_schema["id"], return_to)
+
+        self._rule_evaluator = RuleEvaluator(
```

Comment:
Missing type hints

Reply:
Not enforced in this file, so won't be validated.